MetaMask/metamask-extension
# fix: update stake image

## **Description**

Updates the stake announcement image to a more up-to-date version to match the actual row.

## **Manual testing steps**

1. Install MM build with this change
2. The Stake announcement should appear, either last on the "What's New" list if you haven't seen previous announcements or as the only item in "What's New" if you have
3. The stake announcement should use the updated image

## **Screenshots/Recordings**

### **Before**

Stake announcement preview image uses outdated design:
<img width="314" alt="Screenshot 2024-03-05 at 12 05 41 PM" src="https://github.com/MetaMask/metamask-extension/assets/15035587/6608e5c7-f15e-44d5-8209-f398a35eae6f">

### **After**

Stake announcement preview image matches actual asset list item:
<img width="322" alt="Screenshot 2024-03-05 at 3 17 59 PM" src="https://github.com/MetaMask/metamask-extension/assets/15035587/0d44ec7d-46b6-42e4-83c6-89bdbf4c9594">


## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
